bakerydemo/base/blocks.py

```diff
@@ -77,3 +94,12 @@ class BaseStreamBlock(StreamBlock):
         icon="media",
         template="blocks/embed_block.html",
     )
+
+
+# StreamBlocks for Details
+class DetailsStreamBlock(StreamBlock):
```

Comment:
I’m really surprised we’re not adding this to `BaseStreamBlock` – this feels like a clear missed opportunity to demonstrate Wagtail’s flexibility, by allowing series of "details" content to be placed at arbitrary points?